Lecture-4/Task2/l4task2/Application.java

```diff
@@ -0,0 +1,42 @@
+package l4task2;
+
+import java.time.DayOfWeek;
+import java.util.Scanner;
+
+public class Application {
+
+    private static void printInfo(int dayNumber){
+        System.out.println(DayOfWeek.of(dayNumber));
+    }
+
+    public static void main(String[] args) {
+        System.out.println("type the day number: ");
+        Scanner scanner = new Scanner(System.in);
+        int dayNumber = scanner.nextInt();
+        switch (dayNumber) {
+            case 1:
+                printInfo(dayNumber);
```

Comment:
1) `switch` is not a good option here since all valid values lead to the same code path. See how many duplicated lines are there. `If-else` is much better. 
2) Moreover, if you use some API like `DayOfWeek` - you should study it's docs:  https://docs.oracle.com/javase/8/docs/api/java/time/DayOfWeek.html#of-int- and see that method automatically handles invalid `int` values. So, no need to check it manually in `switch` statement

Reply:
done